flash_update/serializers.py

```diff
@@ -18,7 +18,10 @@
     FlashCountryDistrict,
     FlashGraphicMap,
     FlashAction,
-    FlashActionsTaken
+    FlashActionsTaken,
+    DonorGroup,
+    Donors,
+    FlashUpdateShare
```

Comment:
Let's always add `,` for the last item in the list as well. This creates a better diff for future changes.